prime-router/docker-compose.yml

```diff
@@ -23,6 +23,8 @@ services:
       - AZ_PHD__ELR__PASS=pass
       - AZ_PHD__ELR_TEST__USER=foo
       - AZ_PHD__ELR_TEST__PASS=pass
+      - AZ_PHD__ELR_HL7_TEST__USER=foo
+      - AZ_PHD__ELR_HL7_TEST__PASS=pass
       - AZ_PHD__ELR_PROD__USER=foo
```

Comment:
Not elegant, but easy to do.

OVERALL NOTE:  The vast majority of these changes in the PR are just glop generation of updated schema documentation.   You could argue either way, whether these generated files do or do not belong in git.